.travis.yml

```diff
@@ -60,28 +60,21 @@ env:
     - "GEM=ac:integration"
 
 rvm:
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
+  - 2.4
```

Comment:
@jeremy  this means `2.4.1`... is that what you meant?

Reply:
Nope. That's disappointing.

Reply:
Flipped back to major.minor.tiny @ 6b3acf151829ee16a946c0d77529d236f115234c 🙃